### PR TITLE
Update connector with the new update config event

### PR DESCRIPTION
**Pull request by Igoraamc**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [x] Other, please describe:
         Update config event.

**Does this PR introduce any breaking changes?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] Related issues are referenced in the PR (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] All tests are passing
- [x] New/updated tests are included

**Testing environment:**

- Operating System/Platform: macOS Catalina
- Node version: v12.16.0
- NPM version: 6.13.4

### CODE REVIEW

File: src/interactors/interactors.test.js [hunk 1]
```diff
@@ -10,22 +9,21 @@ jest.mock('@cesarbr/knot-fog-connector-knot-cloud');
...
+        name: 'bool-sensor',
+      },
+      event: {
+        change: true,
+        timeSec: 10,
+        lowerThreshold: 1000,
+        upperThreshold: 3000,
+      },
     },
   ],
```
Comment (esdrasjnr) [inline src/interactors/interactors.test.js]: `sensorId` should be inside the `config` list. Note that the config field hold the list of thing's sensors (and actuators) and its configurations, including the ID (denoted as `sensorId`).

**Review by asf89 — APPROVED** [review]
Great patch! 😃

**Review by esdrasjnr — CHANGES_REQUESTED** [review]
Great, just left one comment I think that should be fixed 👍🏻 Let me know if you have any questions about it 😃

File: src/interactors/interactors.test.js (hunk 1 above)
Comment (Igoraamc) [reply src/interactors/interactors.test.js]: Yeah, that was my bad!

Done!

**Review by esdrasjnr — APPROVED** [review]
Excellent! 💯

**Review by CaioCordeiro — APPROVED** [review]
LGTM! Great Patch! 😄

**Review by netoax — APPROVED** [review]
LGTM!

**Pull request merged**